src/precompile_utils.c

```diff
@@ -382,7 +382,14 @@ static void *jl_precompile_trimmed(size_t world)
             jl_array_ptr_1d_push(m, ccallable);
     }
 
-    void *native_code = jl_create_native(m, NULL, jl_options.trim, 0, world);
+    void *native_code = NULL;
+    JL_TRY {
+        native_code = jl_create_native(m, NULL, jl_options.trim, 0, world);
+    } JL_CATCH {
+        // The verification check (presumably) failed. The error message should
```

Comment:
We should do something to make sure it was actually the verification failing that caused the exception.